_posts/2023-08-10-save-the-date-open-board-meeting.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Sept 21st Bridge Foundry Board and Community Meeting!
+tags: ready
+---
+
+<a href="https://docs.google.com/forms/d/e/1FAIpQLSci1xERIDlSgvb5KCKBBuRX6X0blgviyLES4-qgsAeGe12tAg/viewform"><img src="{{ site.github.url }}/images/blog/2023-08-10-save-the-date.png" alt="invite bridge foundry sept meeting" width="100%"/></a>
+We (the Bridge Foundry board) would like to invite all members of our community to join us as we look back and celebrate all that we have accomplished since our initial strategic plan in 2017 and, with your help, start planning our future. We’ve done a lot over the past six years, and it’s important to come together as a community to celebrate our accomplishments while we also looking ahead to the exciting work we have in front of us. Join for a celebratory session and help us start planing to build an even brighter future for our community.
```

Comment:
```suggestion
We (the Bridge Foundry board) would like to invite all members of our community to join us as we look back and celebrate all that we have accomplished since our initial strategic plan in 2017 and, with your help, start planning our future. We’ve done a lot over the past six years, and it’s important to come together as a community to celebrate our accomplishments while we also looking ahead to the exciting work we have in front of us. Join for a celebratory session and help us start planning to build an even brighter future for our community.
```